linetools/spectra/tests/test_xspec_utils.py

```diff
@@ -111,11 +111,11 @@ def test_addnoise(spec):
 def test_box_smooth(spec):
     # Smooth
     newspec3 = spec.box_smooth(3)
-    np.testing.assert_allclose(newspec3.flux[4000], 0.9650185, rtol=1e-5)
+    np.testing.assert_allclose(newspec3.flux[4000], 0.8429098, rtol=1e-5)
     assert newspec3.flux.unit == u.dimensionless_unscaled
 
     newspec5 = spec.box_smooth(5)
-    np.testing.assert_allclose(newspec5.flux[3000], 1.0405008,rtol=1e-5)
+    np.testing.assert_allclose(newspec5.flux[3000], 0.7609909,rtol=1e-5)
     # Preserve
     newspec5p = spec.box_smooth(5, preserve=True)
 
```

Comment:
I don't think there really was a bug.  Or I can't recall.
Main issue was that the scipy rebin method was no longer working
because of an update in Python.